bundler/bundle_manager.py

```diff
@@ -45,19 +45,19 @@ def __init__(
         self.chain_id = chain_id
         self.is_send_raw_transaction_conditional = is_send_raw_transaction_conditional
 
-    async def send_next_bundle(self) -> None:
+    async def send_next_bundle(self, is_unsafe: bool, is_optimism_gas_estimation: bool) -> None:
```

Comment:
configuration values like is_unsafe and is_optimism_gas_estimation should be stored as a class property instead of passing them as a parameter to class methods